scripts/stac/imagery/collection.py

```diff
@@ -40,8 +44,18 @@ def __init__(
             "license": "CC-BY-4.0",
             "links": [{"rel": "self", "href": "./collection.json", "type": "application/json"}],
             "providers": [],
+            "linz:lifecycle": metadata["lifecycle"],
+            "linz:geospatial_category": metadata["category"],
+            "linz:region": metadata["region"],
+            "linz:security_classification": "unclassified",
         }
 
+        # Optional metadata
+        if metadata.get("event_name"):
```

Comment:
To avoid a duplicate lookup you could use `if event_name := metadata.get("event_name"):`. Ditto below.